client/packages/common/src/ui/components/errors/ErrorWithDetails.tsx

```diff
@@ -14,17 +14,16 @@ export type ErrorWithDetailsProps = {
 export const ErrorWithDetails: React.FC<ErrorWithDetailsProps> = ({
   error,
   details,
-}) => {
-  const n = useNotification();
-
-  return (
-    <Box display="flex" sx={{ color: 'error.main' }} gap={1}>
-      <Box>
-        <AlertIcon onClick={n.error(details)} />
-      </Box>
-      <Box>
-        <Typography sx={{ color: 'inherit' }}>{error}</Typography>
-      </Box>
+}) => (
+  <Box display="flex" sx={{ color: 'error.main' }} gap={1} padding={2}>
+    <Box>
+      <AlertIcon />
     </Box>
-  );
-};
+    <Box>
+      <Typography sx={{ color: 'inherit' }}>{error}</Typography>
+    </Box>
+    <Box>
+      <InfoTooltipIcon title={details} />
```

Comment:
I guess another way would be to show Incorrect password as `link` and the whole component has the on hover

Reply:
that would look better, yeah. I like having the i icon though, it gives a good hint that you might be able to hover or click it or something